src/pathwayvis/components/legend/legend.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="legend" md-whiteframe="4" layout="column">
+	<md-list>
+		<md-list-item>
+			<div layout="column" style="width: 100%">
+				<span class="md-body-1">Flux</span>
+				<div class="grad"></div>
+				<div layout="row">
+					<span class="md-caption">min</span>
+					<span class="md-caption right-span">max</span>
+				</div>
+			</div>
+		</md-list-item>
+		<md-list-item>
+			<div layout="row">
+				<div class="color empty"></div>
+				<span class="md-caption"
+					  layout-align="center center"
+				>
+					No reaction data
+				</span>
+			</div>
+		</md-list-item>
+		<md-list-item>
+			<div layout="row">
+				<div class="color error"></div>
+				<span class="md-caption legend-text"
+					  layout-align="center center"
+				>
+					Error in model
```

Comment:
Red means that reaction is on the map but not in the model, the model is still correct though. "Not in the model" I guess